draft-ietf-rats-corim.md

```diff
@@ -2233,7 +2291,7 @@ The comparison MUST return false if there are no hash algorithms from the condit
 
 ##### Comparison for raw-value entries
 
-> [Andy] *I think this comparison method only works if the entry is at key 4 (because
+> [^issue] *I think this comparison method only works if the entry is at key 4 (because
 there needs to be a mask at key 5). Should we have a Reference Value of this
 which stores `[expect-raw-value raw-value-mask]` in an array?*
```

Comment:
This line can be removed.  The section is still empty, it's unclear what this text refers to.

```suggestion
```